packages/es-components/src/components/controls/buttons/DropdownButton.js

```diff
@@ -47,7 +47,7 @@ const ButtonPanel = styled.div`
   display: ${props => (props.isOpen ? 'block' : 'none')};
   margin-top: 3px;
   position: relative;
-  z-index: 999;
+  z-index: 1000;
```

Comment:
Should we be changing the z-index in es-component? This will effect EVERY other project that used the DropdownButton component.